.github/workflows/XmsCore-CI.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         platform: [macos-10.15]
-        python-version: [3.6]
+        python-version: ['3.10']

```

Comment:
Is there a reason we went with quotes here? Doesn't bother me, just curious.

Reply:
Without the quotes, it tries to use python 3.1 for some reason.